tests/datasource/fluent/integration/conftest.py

```diff
@@ -100,22 +100,28 @@ def pandas_data(
 
 
 def sqlite_datasource(
-    context: AbstractDataContext, db_filename: str
+    context: AbstractDataContext, db_filename: str | pathlib.Path
 ) -> SqliteDatasource:
-    relative_path = pathlib.Path(
-        "..",
-        "..",
-        "..",
-        "test_sets",
-        "taxi_yellow_tripdata_samples",
-        "sqlite",
-        db_filename,
-    )
-    db_file = pathlib.Path(__file__).parent.joinpath(relative_path).resolve(strict=True)
+    db_file_path = pathlib.Path(db_filename)
+    if db_file_path.is_absolute():
```

Comment:
Why do we need this new branch?

Reply:
I don't think we do. It was because the dB I wanted was at a different level. But I think I can just include  `../..` in my path without changing this.